cmake/VASTConfigVersion.cmake

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.14...3.19 FATAL_ERROR)
 
-set(VAST_VERSION_FALLBACK "2021.03.25-rc1-0-")
+set(VAST_VERSION_FALLBACK "2021.03.25-rc2-0-")
```

Comment:
Mind adding a comment that the trailing slash is intended? For sanity. :-)

Reply:
Let's not do this here, but rather in https://github.com/tenzir/vast/pull/1482